src/routes/clients.ts

```diff
@@ -4,9 +4,9 @@ import controller from '../controller/clients';
 const router = express.Router();
 
 router.get('/clients', controller.getClients);
-router.get('/clients/:id', controller.getClient);
-router.put('/clients/:id', controller.updateClient);
-router.delete('/clients/:id', controller.deleteClient);
-router.post('/clients', controller.addClient);
+router.get('/client/:id', controller.getClient);
+router.put('/updateClient/:id', controller.updateClient);
+router.delete('/deleteClient/:id', controller.deleteClient);
+router.post('/addClients', controller.addClient);
```

Comment:
```suggestion
router.get('/client/:id', controller.getClient);
router.put('/client/:id', controller.updateClient);
router.delete('/client/:id', controller.deleteClient);
router.post('/client', controller.addClient);
```

referenced by https://github.com/tm-pe2/ad-api/pull/13/files#r829762041